pkg/directory/directory_windows.go

```diff
@@ -5,7 +5,6 @@ package directory
 import (
 	"os"
 	"path/filepath"
-	"strings"
```

Comment:
yeah--my vim plugins refuse to allow unused package to remain in the imports; I thought all the golint, gofmt stuff catches that, so not sure how it was checked in without using "strings" package at all?